static/scss/footer.scss

```diff
@@ -71,15 +61,26 @@
 }
 
 .footer-bottom {
-  padding-top: 20px;
+  padding-top: 40px;
   padding-bottom: 20px;
+  background: $white-smoke-dark;
+  padding-left: 999px;
```

Comment:
Can you include a comment about why these padding/margin definitions are necessary?

Reply:
That's for carrying the footer background to the corners on both left and right sides.

Reply:
Try to remove that it and test it on your system. You will get it pretty easily.

Reply:
If we need it, please include that as a comment in the code

Reply:
@umarmughal824 If you're just trying to make sure the background of the `footer-bottom` element stretches to the edge, you can just add a `p-0` class to the `container` element in the footer, then add normal padding to `footer-bottom`. Please try that out and let me know if that doesn't work

Reply:
And the reason of giving the big number is that It should not break on bigger screens.

Reply:
sure, let me try it.

Reply:
that's not going to work what are you suggesting.
on bigger screens, it would not work. I just tested in on the left side.
<img width="1401" alt="Screenshot 2020-06-26 at 19 51 52" src="https://user-images.githubusercontent.com/4043989/85870533-934ed680-b7e6-11ea-9b88-3476b76088f1.png">

Reply:
this is how it currently looks like on that big size

<img width="1434" alt="Screenshot 2020-06-26 at 19 55 48" src="https://user-images.githubusercontent.com/4043989/85870881-16702c80-b7e7-11ea-8a28-5244ba1b620b.png">